glide.yaml

```diff
@@ -7,5 +7,6 @@ import:
 - package: github.com/dgrijalva/jwt-go
   version: v3.2.0
 - package: github.com/edgexfoundry/edgex-go
+  version: delhi
```

Comment:
This is wrong, we want the master version of this repository to build from the master version of edgex-go.

Reply:
@tingyuz this needs to be resolved before this branch can be merged back to master, otherwise we run the risk of not having the master branch of security-api-gateway follow the master branch of edgex-go.